bokeh/models/mappers.py

```diff
@@ -5,10 +5,9 @@
 
 from ..model import Model
 from ..core.properties import abstract
-from ..core.properties import Float, Color, Enum, Seq, Instance
+from ..core.properties import Float, Color, Enum, Seq
 from ..core.enums import Palette
 from .. import palettes
-from .transforms import Jitter
```

Comment:
This is causing the test problem. The example does `from bokeh.models import Jitter` which was only "working: because this import was transitively available in `bokeh.models` @Maggie-M is going to make another issue/PR to fix, then you can merge that into this PR and it will pass.